### DISCUSSION TITLE
For terraos, do I need to recompile systemd for a linux distro like ubuntu to get it to run?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1362] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am asking because if you use shimboot you have to patch systemd, but terraos just says to bootstrap it into the ext4 partition, nothing about recompiling systemd.

**Comment by Charlieis1nsan3** [comment]
yes terraos is built off of Shimboot, and I'm pretty sure you need a compiled shim (I think)  just download your shim and build it here. https://darkn.bio/sh1mmer/builder    and then once your injected bin is complete get your second compute or Chromebook and download the chrome recovery app, and boom

**Comment by FunnyCorgi** [comment]
I don't think that's it, I am asking if I have to recompile the system hardware driver daemon to work with the chromebook board hardware, basically take the one from the shim and make it work with the ubuntu(which is what shimboot does, but I don't know if I need to do this for ubuntu).

**Reply by Charlieis1nsan3** [reply]
nope. the only thing that needs to be fixed, is the networking. shims are basically prebuilt ubuntu's

**Comment by S-PScripts** [comment, marked as answer]
yarl
